nomad/rust-integration-tests.nomad

```diff
@@ -92,6 +100,10 @@ variable "plugin_work_queue_db" {
 
 locals {
   dns_servers = [attr.unique.network.ip-address]
+
+  # Currently we use the same version for all containers. As such we pick one container to get the version from
+  app_version                      = split(":", var.container_images["rust-integration-tests"])[1]
```

Comment:
What, specifically, does `app_version` mean? What will the impact be when the version is not the same for all containers?

Note that we _don't_ actually use the same version for all containers. If a container image hasn't changed, it shouldn't be built and uploaded again, meaning that we can have a number of different versions in play at the same time.

Reply:
This is a pattern I carried over from the rest of our Nomad files; perhaps ping Thomas about it?

![image](https://user-images.githubusercontent.com/69007229/198109797-673d41d3-498c-44d7-95c6-82b86ac9ce39.png)

Reply:
Ping @twunderlich-grapl ☝️

Reply:
sorry, i actually copy pasted that comment but since there's only one container here then that comment doesnt apply

Reply:
@christophermaier,
This currently assumes that the version tag is the same for the entire job, since I thought we were retagging all the containers with the same version. If that's incorrect, we just need to move the app_version split statement into each job.

Reply:
I can go through and update that in a follow-up PR